coalesce/volunteers/migrations/0002_auto_20210420_1138.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.8 on 2021-04-20 11:38
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('volunteers', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='volunteer',
+            name='organizer_comments',
+            field=models.TextField(help_text='Notes from organizers about this volunteer'),
+        ),
```

Comment:
This change to volunteers.models.Volunteer was not made in your pull request. Looks like this migration was accidentally forgotten in a previous pull request. It can be included here or as a separate commit.

Reply:
Happy to leave it here or revert, your call